.github/workflows/proto.yml

```
@@ -75,3 +75,26 @@ jobs:
           BASE_BRANCH="..origin/${{ steps.setup.outputs.base-branch }}"
           export BASE_BRANCH
           make proto-update-check
+      - name: Proto-format check
+      run: |
+        make proto-format
+        if [ -n "$( git status --porcelain )" ]; then
+          git diff
+          exit 1
+        fi
+      - name: Proto-gen check
+      run: |
+        make proto-gen
+        if [ -n "$( git status --porcelain )" ]; then
+          git diff
+          exit 1
+        fi
+      - name: Swagger update check
+          run: |
+            make update-swagger-docs
+            if [ -n "$( git status --porcelain )" ]; then
+              git diff
+              exit 1
+            fi
```

Indation isn't right here. The first two aren't tabbed in enough and the last one is tabbed in too much.

```suggestion
      - name: Proto-format check
        run: |
          make proto-format
          if [ -n "$( git status --porcelain )" ]; then
            git diff
            exit 1
          fi
      - name: Proto-gen check
        run: |
          make proto-gen
          if [ -n "$( git status --porcelain )" ]; then
            git diff
            exit 1
          fi
      - name: Swagger update check
        run: |
          make update-swagger-docs
          if [ -n "$( git status --porcelain )" ]; then
            git diff
            exit 1
          fi
```

Additionally, I don't think the github runners have docker installed, and all three of those `make` targets require it. So I don't think they'll work even with the correct indentation.